### PR TITLE
[Calendar] add declarative calendar skill

### PR DESCRIPTION
**Pull request by KayMKM**
### Purpose
*What is the context of this pull request? Why is it being done?*
add declarative calendar skill

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/declarative/calendarskill/generated/luis.settings.composer.westus.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "luis": {
+        "CreateMeetingDialog_en-us_lu": "3d0cf371-83c9-4b56-aee6-86511771077b",
```
Comment (xieofxie) [inline skills/declarative/calendarskill/generated/luis.settings.composer.westus.json:3]: remove generated

**Comment by xieofxie** [comment]
Declarative are hard to review, sigh..

File: skills/declarative/calendarskill/generated/luis.settings.composer.westus.json (hunk 1 above)
Comment (KayMKM) [reply skills/declarative/calendarskill/generated/luis.settings.composer.westus.json]: removed

File: skills/declarative/calendarskill/dialogs/ShowMeetingDialog/ShowMeetingDialog.dialog [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+              },
+              "condition": "dialog.ShowMeeting_ToTime == null",
+              "actions": [
+                {
+                  "$kind": "Microsoft.SetProperty",
+                  "$designer": {
+                    "id": "Jlf29u"
+                  },
+                  "property": "dialog.ShowMeeting_ToTime",
+                  "value": "=formatDateTime('03/15/2018 23:59:59', 'yyyy-MM-ddTHH:mm:ss')"
```
Comment (xhr0804) [inline skills/declarative/calendarskill/dialogs/ShowMeetingDialog/ShowMeetingDialog.dialog:142]: Why use this constant value, should we use a prompt to ask user for the time?

Comment (KayMKM) [reply skills/declarative/calendarskill/dialogs/ShowMeetingDialog/ShowMeetingDialog.dialog:142]: If user didn't provide end time, we will use a end of day time stamp as default value.

**Comment by xhr0804** [comment]
@bobokids The current features of the flow is Very basic, more efforts are needed to improve the robustness and availability, do you think we can first check it in, and optimize it later?

**Pull request merged**